src/linuxthreads.c

```diff
@@ -597,10 +598,10 @@ int ListAllProcessThreads(void *parameter,
       #define SYS_ERRNO        my_errno
       #define SYS_INLINE       inline
       #define SYS_PREFIX       0
-      #undef  SYS_LINUX_SYSCALL_SUPPORT_H
-      #include "linux_syscall_support.h"
+      // #undef  SYS_LINUX_SYSCALL_SUPPORT_H
+      // #include "linux_syscall_support.h"
```

Comment:
If this is not needed why don't we just remove it?

Was it ever needed?  If it was before, why is it not needed now?

Reply:
I have fixed the error "redefinition of ‘struct kernel_dirent64’". and these lines are not commented out anymore.
I will update the commit message again.